src/odemis/acq/stitching/_constants.py

```diff
@@ -0,0 +1,19 @@
+REGISTER_IDENTITY = 0
+REGISTER_SHIFT = 1
+REGISTER_GLOBAL_SHIFT = 2
+WEAVER_MEAN = 0
+WEAVER_COLLAGE = 1
+WEAVER_COLLAGE_REVERSE = 2
+
+GOOD_MATCH = 0.9  # consider all registrations with match > GOOD_MATCH
+LEFT_TO_RIGHT = 1
+RIGHT_TO_LEFT = -1
+
+# TODO: Find a value that works fine with cryo-secom
+# Ratio of the allowed difference of tile focus from good focus
+# The bigger, the less likely the autofocus will be triggered
+FOCUS_FIDELITY = 0.3
+# Limit focus range, half the margin will be used on each side of initial focus
+FOCUS_RANGE_MARGIN = 100e-6  # m
+# Indicate the number of tiles to skip during focus adjustment
+SKIP_TILES = 3
```

Comment:
No, this shouldn't be all the constants. It should be only the constants used by external callers (ie, constants used in the module interface). 
So the constants that are used to tweak the internal module behaviour, or just to make the code more readable shouldn't appear here, but just be in the submodule that they help. (ie, every constant starting from l. 8 should go back).

Reply:
Thanks for the clarification, returned these constants to their submodules.